cmd/edit/machinepool/cmd.go

```diff
@@ -210,13 +232,37 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 	if cluster.MultiAZ() && replicas%3 != 0 {
 		reporter.Errorf("Multi AZ clusters require that the number of machine pool replicas be a multiple of 3")
+=======
```

Comment:
Conflict market - might want to check what's changed...

Reply:
fixed